s3/src/test/scala/akka/stream/alpakka/s3/scaladsl/S3SourceSpec.scala

```diff
@@ -36,7 +36,7 @@ class S3SourceSpec extends S3WireMockBase with S3ClientIntegrationSpec {
     mockDownload()
 
     //#download
-    val (s3Source: Source[ByteString, _], _) = s3Client.download(bucket, bucketKey)
+    val Some((s3Source: Source[ByteString, _], _)) = s3Client.download(bucket, bucketKey).futureValue
```

Comment:
This code is part of the documentation and will look rather strange there. I guess this needs to be rewritten to show how to use the changed API.
(applies to the other snippets, as well)

Reply:
Hi! I tried to make this snippets consistent with the `getObjectMetadata` snippets. Can you have a look?